src/status_im/ens/core.cljs

```diff
@@ -225,6 +226,12 @@
       :searching)
     :else :invalid))
 
+(defn addresses-without-watch
+  [db]
+  (into #{}
+        (remove #(= (:type %) :watch)
+                (map #(eip55/address->checksum (:address %)) (get db :profile/wallet-accounts)))))
```

Comment:
Are we sure this won't be lazy and that it'll go through of the elements?

Reply:
i'm just moving stuff

Reply:
this is old code, so we shouldn't care much